### PR TITLE
SLURM as alternative resource manager

### LINKED ISSUE #305: Slurm/PBS support
Resources of many HPC clusters are managed by slurm or PBS. We should provide support for those managers in place of default redis stack.

This is of high priority for new apoz cluster architecture.

### PR DESCRIPTION
**Pull request by kowalskiw**
This PR is to introduce SLURM support for AAMKS queuing and task management.

Resolves #305

### CODE REVIEW

**Comment by kowalskiw** [comment]
Additionally I removed not used anymore `dev` directory. For two years we use branches instead.

File: montecarlo/cfast_mcarlo.py [hunk 1]
```diff
@@ -94,10 +94,15 @@ def _cfast_record(self, key, name=None):
             return table
 
     def read_json(self):
+        print(os.environ['AAMKS_PROJECT'])
```
Comment (mmackiewicz97) [inline montecarlo/cfast_mcarlo.py:97]: probably no longer needed

File: montecarlo/cfast_mcarlo.py [hunk 2]
```diff
@@ -410,18 +415,23 @@ def __init__(self, sim_id):
...
-        self.config = self.json.read(os.path.join(os.environ['AAMKS_PATH'], 'evac', 'config.json'))
+        self.config = self.json.read(os.path.join(os.environ['AAMKS_PATH'], 'aamks', 'evac', 'config.json'))
         self._scen_fuel = {}
         self._fire = None
         self._fires = []
         self._fire_openings = []
         self.sections = {}
 
     def __read_json(self):
+        [print(i, j) for i,j in os.environ.items()]
```
Comment (mmackiewicz97) [inline montecarlo/cfast_mcarlo.py:426]: as previous

**Review by mmackiewicz97 — APPROVED** [review]
looks good and cleaner

**Pull request merged**